test/addons-napi/test_handle_scope/test.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+const common = require('../../common');
+const assert = require('assert');
+
+// testing handle scope api calls
+const test_handle_scope =
```

Comment:
sure just copied other existing test as a base, but that's not a good reason :).  Updating